packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -507,6 +531,64 @@ export class GrpcPlugin extends BasePlugin {
       metadata.set(GRPC_TRACE_KEY, serializedSpanContext);
     }
   }
+
+  /** Method to record stats for client and server. */
+  static recordStats(
+      // tslint:disable-next-line:no-any
+      kind: SpanKind, tags: TagMap, argsOrValue: any, reqOrRes: any,
```

Comment:
Could you use the empty object type `{}` in place of `any` here? (That should work because you are not actually referencing anything about the type, just transparently passing it into `sizeof`)

Reply:
SGTM, done